### PR TITLE
Alert on FullDisruption state for NodeController

### PR DESCRIPTION
**Pull request by kokaz**
This changes the configuration of Elastalert so it can load multiple rules. This also add a rule for `FullDisruption` that might occurs on NodeController.

### CODE REVIEW

File: deployment/rules/rules.template.yaml [hunk 1]
```diff
@@ -27,4 +27,25 @@ data:
...
+  zone-fulldisruption.yaml: |-
+    es_host: ${ES_HOST}
+    es_port: 443
+    use_ssl: true
+    name: zone-fulldisruption
+    type: frequency
+    index: logstash-*
+    num_events: 1
+    timeframe:
+        minutes: 5
```
Comment (jwolski2) [inline deployment/rules/rules.template.yaml]: What does the timeframe / minutes value represent? If it represents how soon an alert will be triggered. 5 mins is much too high. It should be 1 minute. We should strive for our SRE time-to-mitigation SLA somewhere under 10 mins.

Comment (kokaz) [inline deployment/rules/rules.template.yaml]: I guess it's that, I just copied the other rule but will change to something sooner

**Comment by kokaz** [comment]
@jwolski2 This doesn't post to PagerDuty though so not sure will be paged for that. Also I will deploy to staging first and see if it works

**Comment by jwolski2** [comment]
@kokaz sounds good! That's lame about not going to pagerduty. I'll add a backlog issue for that.

**Comment by faiq** [comment]
so this will alert first time we see this in a 1 minute span?

**Comment by kokaz** [comment]
Proof of work

image

**Pull request merged**